.travis.yml

```diff
@@ -55,41 +55,45 @@ matrix:
       rust: stable
     - os: osx
       rust: nightly
+    - os: osx
+      env: TARGET=aarch64-apple-ios
+      rust: stable
+      dist: trusty
 script:
   - rustc --version
   - sysctl -a | grep mem
   - if [[ "$TRAVIS_RUST_VERSION" == "nightly" ]]; then
-      (rustup component add clippy && cargo clippy) || touch clippy_install_failed;
+    (rustup component add clippy && cargo clippy) || touch clippy_install_failed;
```

Comment:
Please don't change this indent, I made it to make it appear more obvious that it's inside the `if`.

Reply:
my bad, I guess the IDE did that

Reply:
I think so, you added the missing backline at the end too (and that's nice :smiley: ).